src/middleware/backend-authentication.js

```diff
@@ -14,43 +24,61 @@ module.exports = (app, appName) => {
 	if (!backendKeys.length) {
 		const missingEnvVarsMessage = [
 			'Backend authentication is disabled due to missing environment variables -',
-			'If backend authentication isn\'t required then disable this middleware by setting the n-express \'withBackendAuthentication\' option to false.',
+			"If backend authentication isn't required then disable this middleware by setting the n-express 'withBackendAuthentication' option to false.",
 			'To enable backend authentication add the missing environment variables.',
 			'For more info on backend authentication see https://github.com/Financial-Times/n-express#optional',
 		];
```

Comment:
**suggestion**: why not go further and make the whole thing a template string

Reply:
This was a Prettier change tbf, not really related to this PR's topic. See the comment I made below